workbench-core/accounts/src/models/createCostCenter.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+interface CreateCostCenter {
+  name: string;
+  dependency: string;
```

Comment:
If dependency doesn't make sense externally, why are we using it internally?

Reply:
@nguyen102 Added a question on Quip about this.

Reply:
This make sense, let's change it to `accountId`. The request doesn't have to exactly match how we store the data in DDB. I'll update the design doc.

Reply:
Hmm, this is interesting idea. We haven't added a model for create requests before. @maghirardelli , @FernandoAranda , @kpark277 thoughts on having a request interface for all create requests? 

So in this case, this would be named `createCostCenterRequest`.